lib/marginalia/sidekiq_instrumentation.rb

```diff
@@ -13,7 +13,28 @@ def call(worker, msg, queue)
       end
     end
 
-    def self.enable!
+    def self.enable!(annotate_delayed_class_extension: false)
+      if annotate_delayed_class_extension
+        Sidekiq::Extensions::DelayedClass.class_eval do
+          # NOTE: Prefer redefining perform so that YAML is only parsed once.
+          # Adding a Marginalia::Comment.components would incur an extra parsing on every query
+          # in a delayed class
+          def perform(yml)
```

Comment:
@bf4 Do you have a shareable sample or example Yaml block that would be used as the input here? Is this a Sidekiq thing that has serialized values for target, method_name, and args?